README.md

```diff
@@ -46,8 +46,7 @@ obs.RequestName({args}, callback(err, data)) returns Promise
 obs.requestName({args}, callback(err, data)) returns Promise
 
 // The following are additional supported requests.
-obs.connect({ address: 'address' }, callback(err, data)) returns Promise
-obs.authenticate({ password: 'password' }, callback(err, data)) returns Promise
+obs.connect({ address: 'address', password: 'password' }, callback(err, data)) returns Promise
```

Comment:
Sure, that works. 
I might close the connection then if authentication fails such that there's hopefully no awkward disconnect/reconnect stuff going on. I'll add that at some point.